### PR TITLE
Improve rebrand styling

### PR DESCRIPTION
**Pull request by fabric-8**
Various design improvements touching the home page and stacking details pages, including typography, spacing and colors

### CODE REVIEW

File: src/pages/choose-stacking-method/choose-stacking-method.layout.tsx [hunk 1]
```diff
@@ -83,7 +84,7 @@ In return, you are eligible to receive rewards in the form of BTC"
         description="If you meet the protocol minimum, you can Stack your STX independently by directly interacting with the protocol. You also have the option to delegate your STX to a stacking pool provider."
         image={<EarnWithStackingImage />}
       />
-      <StackingOptionsCardContainer>
+      <StackingOptionsCardContainer background="ink.background-primary">
```
Comment (camerow) [inline src/pages/choose-stacking-method/choose-stacking-method.layout.tsx:87]: If you want that bg to apply to all uses of that layout you can add it to `StackingOptionsCardContainer`'s root element

**Review by camerow — APPROVED** [review]

File: src/pages/choose-stacking-method/choose-stacking-method.layout.tsx (hunk 1 above)
Comment (fabric-8) [reply src/pages/choose-stacking-method/choose-stacking-method.layout.tsx:92]: thx & yep, fixed!

File: src/components/navbar.tsx [hunk 2]
```diff
@@ -67,28 +106,33 @@ const NavbarLeft = () => {
...
+      gap="space.05"
+      justifyItems={'flex-end'}
+    >
+      <NavItem href="https://earn.leather.io/" isActive>
+        Earn
+      </NavItem>
+      <NavItem href="https://leather.io/blog">Blog</NavItem>
+      <NavItem href="https://leather.io/learn">Learn</NavItem>
+      <NavItem href="https://leather.io/guides">Guides</NavItem>
+      <NavItem href="https://leather.io/developer">Developer docs</NavItem>
```
Comment (camerow) [inline src/components/navbar.tsx:126]: This should be `developer-docs`

File: src/components/hr.tsx [hunk 3]
```diff
@@ -1,5 +1,7 @@
 import { Box, BoxProps, color } from '@stacks/ui';
 
 export function Hr(props: BoxProps) {
-  return <Box height="1px" width="100%" backgroundColor={color('border')} {...props} />;
+  return (
+    <Box height="1px" mb="base-loose" width="100%" backgroundColor={color('border')} {...props} />
```
[the remark this answers is not in the document]
Comment (fabric-8) [reply src/components/hr.tsx:5]: You are right, actually no

**Pull request merged**